dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/FATSuite.java

```diff
@@ -78,11 +85,31 @@ public class FATSuite {
         }
     }
 
+    //Server used for setup
+    private static LibertyServer server = LibertyServerFactory.getLibertyServer("globalTLDServer");
+    
+    public static final String USER_FEATURE_PATH = "usr/extension/lib/features/";
+    public static final String USER_BUNDLE_PATH = "usr/extension/lib/";
+    public static final String USER_FEATURE_MF_FAT_PATH = "features/globaltld-1.0.mf";
+    public static final String USER_FEATURE_NAME = "globaltld-1.0.mf";
+    public static final String USER_BUNDLE_JAR_FAT_PATH = "bundles/io.openliberty.test.tld.jar";
+    public static final String USER_BUNDLE_JAR_NAME = "io.openliberty.test.tld.jar";
+
     /**
      * @see {@link FatLogHandler#generateHelpFile()}
      */
     @BeforeClass
-    public static void generateHelpFile() {
+    public static void generateHelpFile() throws Exception {
```

Comment:
Let's rename this method since it is doing more than generating a help file now.